packages/suite/src/utils/suite/storage.ts

```diff
@@ -12,10 +12,14 @@ export const serializeDiscovery = (discovery: Discovery) => ({ ...discovery, run
  * Strip fields from Device
  * @param {AcquiredDevice} device
  */
-export const serializeDevice = (device: AcquiredDevice) => ({
-    ...device,
-    path: '',
-    remember: true,
-    connected: false,
-    buttonRequests: [],
-});
+export const serializeDevice = (device: AcquiredDevice, forceRemember: true | undefined) => {
+    const sd = {
+        ...device,
+        path: '',
+        remember: true,
+        connected: false,
+        buttonRequests: [],
+    };
+    if (forceRemember) sd.forceRemember = true;
```

Comment:
i dont think this is the right way, `device. forceRemember` should be either true or shouldn't be present in device object at all

Reply:
It is not present in the passed device because the parameter passed in the SUITE.REMEMBER action is action.payload, not device after change state.
```
        case SUITE.REMEMBER_DEVICE:
            api.dispatch(
                storageActions.rememberDevice(
                    action.payload,
                    action.remember,
                    action.forceRemember,
                ),
            );
            break;
```
We could change the code in the middleware to `api.getState().devices.find(...))` but I did not want to do it for for me unknown side effects.